packages/input-text/src/protectedValue.js

```diff
@@ -97,19 +114,58 @@ export default WrappedComponent => {
         protected: covered,
         onReplace,
         onKeyDown,
+        onBlur,
+        onChange,
         ...wrappedComponentProps
       } = this.props;
+      const { isEditing } = this.state;
+
       return (
-        <div style={{ position: "relative" }}>
+        <Container isEditing={isEditing}>
           <WrappedComponent
             {...wrappedComponentProps}
             protected={covered}
             focused={this.state.isEditing}
             value={this.state.isEditing ? "" : this.state.value}
             onKeyDown={covered ? this.onKeyDown : onKeyDown}
+            onChange={event => {
+              this.setState({ editedValue: event.target.value });
+              onChange && onChange(event);
+            }}
+            onBlur={event => {
+              covered && isEditing && this.onCancel();
+              onBlur && onBlur(event);
+            }}
           />
           {covered && !this.state.isEditing && this._renderCover()}
-        </div>
+          {covered &&
+            isEditing && (
+              <div>
+                <Button
+                  id="cancel-button"
+                  onClick={this.onCancel}
+                  size={buttonSizes.MEDIUM}
+                  className="margin-button"
+                >
+                  Cancel
```

Comment:
The `Cancel` and `Save` button labels should be customizable.  It's fine for now, given the urgency of this PR, but we'll definitely need this for localization.

Do you think it's quick enough to change now or should we create an issue to handle this later?

Reply:
Good point, I'll do it now